### PR TITLE
Fix GLTFSkin binding for the `godot_skin` property

### LINKED ISSUE #77405: `GLTFSkin` does not report the class_name of the `godot_skin` property.
### Godot version

v4.0.2.stable.official [7a0977ce2]

### System information

Windows 10

### Issue description

The entry for `godot_skin` in `get_property_list` should be:

```
{ "name": "godot_skin", "class_name": &"Skin", "type": 24, "hint": 0, "hint_string": "", "usage": 6 }
```

But is

```
{ "name": "godot_skin", "class_name": &"", "type": 24, "hint": 0, "hint_string": "", "usage": 6 }
```

### Steps to reproduce

Execute this code:

```
	var skin := GLTFSkin.new()
	for property in skin.get_property_list():
		if property["name"] == "godot_skin":
			prints(property)
```

### Minimal reproduction project

[GLTFSkinProperty.zip](https://github.com/godotengine/godot/files/11547845/GLTFSkinProperty.zip)
 (A simple project that executes the code shared above).

### PR DESCRIPTION
**Pull request by theraot**
Fixes https://github.com/godotengine/godot/issues/77405

Tested:
image

*Also tested that reading and writing a GLTF with a rigged model still works. Not depicted.*

### CODE REVIEW

**Review by fire — APPROVED** [review]
@aaronfranke This looks fine?

**Review by aaronfranke — APPROVED** [review]

**Review by fire — CHANGES_REQUESTED** [review]
This needs to be using the resource property hint

File: modules/gltf/structures/gltf_skin.cpp [hunk 1]
```diff
@@ -65,7 +65,7 @@ void GLTFSkin::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "skeleton"), "set_skeleton", "get_skeleton"); // int
 	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "joint_i_to_bone_i", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_INTERNAL), "set_joint_i_to_bone_i", "get_joint_i_to_bone_i"); // RBMap<int,
 	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "joint_i_to_name", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_INTERNAL), "set_joint_i_to_name", "get_joint_i_to_name"); // RBMap<int,
-	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "godot_skin"), "set_godot_skin", "get_godot_skin"); // Ref<Skin>
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "godot_skin", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT, "Skin"), "set_godot_skin", "get_godot_skin"); // Ref<Skin>
```
Comment (fire) [inline modules/gltf/structures/gltf_skin.cpp:68]:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "godot_skin", PROPERTY_HINT_RESOURCE_TYPE, "Skin"), "set_godot_skin", "get_godot_skin");
```

I haven't compiled this but can you check?

Comment (theraot) [reply modules/gltf/structures/gltf_skin.cpp:68]: Tested. Yes, this works.

image

**Review by aaronfranke — APPROVED** [review]

**Review by fire — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**